pkg/querier/frontend/query_range.go

```diff
@@ -127,6 +127,11 @@ func encodeDurationMs(d int64) string {
 	return strconv.FormatFloat(float64(d)/float64(time.Second/time.Millisecond), 'f', -1, 64)
 }
 
+func msToTime(t int64) time.Time {
```

Comment:
I used Prometheus `timestamp.Time()` for this in #1312 - we should be consistent.

Reply:
:+1: